README.md

```diff
@@ -280,18 +280,16 @@ Browse our [Makefile](./Makefile) for a list of available commands such as `make
 
 #### Unit
 
-Unit tests can be ran via invoking `make test-unit`.
+Unit tests can be run with `make test-unit`.
 
-#### Integration / E2E against local deployment
+#### Integration / E2E
 
-Integration tests against op framework can be ran via `make test-e2e-local`. These tests use the [op-e2e](https://github.com/ethereum-optimism/optimism/tree/develop/op-e2e) framework for asserting correct interaction behaviors with batch submission and state derivation.
+Integration tests against op framework can be run with `make test-e2e`. These tests use the [op-e2e](https://github.com/ethereum-optimism/optimism/tree/develop/op-e2e) framework for asserting correct interaction behaviors with batch
```

Comment:
so `test-e2e` runs both the memstore backend as well as holesky backend?

Reply:
Yes, that's correct.

Combining allows test backend to be part of the test matrix, instead of being configured via env vars. I think this could be an issue if holesky tests took a really long time- but since they are quick, I think it makes sense to combine with memstore tests.

Reply:
I'd argue against having them consolidated since holesky tests can fail due to an assortment of issues and rely on external dependency for success

Reply:
hmm I think I agree with ethen here.

Reply:
> holesky tests can fail due to an assortment of issues and rely on external dependency for success

@ethenotethan Fair enough 🤔 

@samlaf You weren't a fan of the env var test configuration, but I'm not sure I see a great alternative it we want to keep memstore vs testnet tests separate. 

What we could do here is build v1 vs v2 testing into the tests themselves, and configure backend with an environment variable, to allow us to separate execution.

Thoughts?

Reply:
Ok, my most recent iteration takes the middle road: 

holesky vs local is decided with an env var, to allow the tests to be easily split into different actions. V1 vs V2 is decided in the test files themselves, though, instead of with an env var